pkg/run/integration_test.go

```diff
@@ -31,6 +31,13 @@ func TestIntegrationSnapshots(t *testing.T) {
 				t.Skip()
 			}
 
+			// Set up environment variables here so we can validate it's
```

Comment:
imo: move this into testExample and get rid of the `name` check (ie: set the env for all tests)

I like that we can understand the whole test environment by reading testExample, and would prefer not to split that over both functions.